code/modules/research/bepis.dm

```diff
@@ -0,0 +1,248 @@
+//This system is designed to act as an in-between for cargo and science, and the first major money sink in the game outside of just buying things from cargo (As of 10/9/19, anyway).
+
+//economics defined values, subject to change should anything be too high or low in practice.
+
+
+/obj/machinery/rnd/bepis
+	name = "\improper B.E.P.I.S. Chamber"
+	desc = "A high fidelity testing device which unlocks the secrets of the known universe using the two most powerful substances available to man: excessive amounts of electricity and capital."
+	icon = 'icons/obj/machines/bepis.dmi'
+	icon_state = "chamber"
+	layer = ABOVE_MOB_LAYER
+	density = TRUE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 1500
+	circuit = /obj/item/circuitboard/machine/bepis
+
+	var/banking_amount = 100
+	var/banked_cash = 0					//stored player cash
+	var/datum/bank_account/account		//payer's account.
+	var/mob/living/carbon/human/H   	//the person using the console in each instance.
+	var/obj/item/card/id/Card   		//the account of the person using the console.
```

Comment:
These two variables do not need to exist, they're only used in ui_act from what i can see.

Reply:
This is still a thing and you added a third one.

Reply:
In case you're not sure how to remove these:
You have access to the user in both ui_act and ui_data, get the data there instead of storing it on the machine.